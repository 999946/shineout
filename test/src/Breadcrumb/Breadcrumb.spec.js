import { mount } from 'enzyme'
import React from 'react'
import { Breadcrumb } from 'shineout'
import Render from 'react-test-renderer'
import BreadcrumbBase from '../../../site/pages/components/Breadcrumb/example-1-base'

describe('Breadcrumb[Base]', () => {
  test('should render correct', () => {
    const data = [{ title: <a href="#/components/Button">Button</a> }, { title: 'Self' }]
    const wrapper = Render.create(<Breadcrumb data={data} />).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
