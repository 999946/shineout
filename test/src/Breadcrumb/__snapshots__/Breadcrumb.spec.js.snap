// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb[Base] should render correct 1`] = `
<div
  className="so-breadcrumb"
  style={Object {}}
>
  <span>
    <a
      href="#/components/Button"
    >
      Button
    </a>
    <span
      className="so-breadcrumb-separator"
    >
      /
    </span>
  </span>
  <span>
    <b>
      Self
    </b>
  </span>
</div>
`;
